internal/apiutil/mfdatabase/connect.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Mainflux
```

Comment:
API util does not look like a good place for this. Something like `internal/db/connect.go`. Also, the most commonly used DB connection (Postgres) is missing.

Reply:
So with Postgres, there is a slight issue. For instance when the auth service connects to Postgres, it will do a migration which is specific to the auth service and hence the bootstrap can't use it.
If we implement a common connect to Postgres this would mean we will import the different modules. Is that okay?